pkg/measurement/measurement.go

```diff
@@ -82,7 +83,11 @@ func InitMeasure(p *properties.Properties) {
 	globalMeasure = new(measurement)
 	globalMeasure.p = p
 	globalMeasure.opMeasurement = make(map[string]ycsb.Measurement, 16)
-	warmUp = 1
+	if p.GetInt64(prop.WarmUpTime, 0) <= 0 {
```

Comment:
please provide an `EnableWarmUp(b bool)` function instead of using global var directly.